analyses/gene-set-enrichment-analysis/run-gsea.sh

```diff
@@ -18,6 +20,8 @@ set -o pipefail
 
 
 IS_CI=${OPENPBTA_TESTING:-0}
+RUN_FOR_SUBTYPING=${OPENPBTA_BASE_SUBTYPING:-0}
+echo $RUN_FOR_SUBTYPING
```

Comment:
This feels like it might be a development remnant, but it also doesn't hurt anything. ¯\\_(ツ)_/¯

Reply:
it's intentional since with the condition we can run gsea with DATA_DIR="../collapse-rnaseq/results" which is updated as part of subtyping https://github.com/kgaonkar6/OpenPBTA-analysis/blob/all_subtyping_gsea_rerun/scripts/run-for-subtyping.sh

Reply:
I was specifically referring to the `echo` line. Since I don't see that elsewhere.